.env.example

```diff
@@ -91,9 +91,9 @@ SESSION_URL_NOTIFICATION_URL="http://localhost/ecommerce/npg/notifications/v1/se
 JWT_NPG_NOTIFICATION_SECRET=ODMzNUZBNTZENDg3NTYyREUyNDhGNDdCRUZDNzI3NDMzMzQwNTFEREZGQ0MyQzA5Mjc1RjY2NTQ1NDk5MDMxNzU5NDc0NUVFMTdDMDhGNzk4Q0Q3RENFMEJBODE1NURDREExNEY2Mzk4QzFEMTU0NTExNjUyMEExMzMwMTdDMDk
 
 JWT_NPG_NOTIFICATION_VALIDITY_TIME=900
-REDIRECT_PAYMENT_TYPE_CODE_LIST=RPIC,RABC
+REDIRECT_PAYMENT_TYPE_CODE_LIST=REDIRECT-RBPR
```

Comment:
[nitpick]
What do you think about removing `REDIRECT-` and using payment type codes directly?

Reply:
The aim of this pr is to modify redirect url mapping as pspId-paymentTypeCode. For mocked environment the specific psp id configured as redirect (gec mock and so on have id REDIRECT). 
So this is now the new mapping :D

Reply:
Maybe here we can remove the "psp" part for the configured keys but then we have less control against configured expected keys